src/dev/build/tasks/os_packages/docker_generator/resources/bin/opensearch-dashboards-docker

```diff
@@ -30,8 +30,10 @@ opensearch_dashboards_vars=(
     opensearch.hosts
     opensearch.logQueries
     opensearch.password
-    opensearch.pingTimeout
     opensearch.requestHeadersWhitelist
+    opensearch.memoryCircuitBreaker.enabled
```

Comment:
If you are adding this here I would make sure to include this here:

https://github.com/opensearch-project/opensearch-build/blob/main/docker/release/config/opensearch-dashboards/opensearch-dashboards-docker-entrypoint.sh (or at least create an issue for it)

Reply:
Done: https://github.com/opensearch-project/opensearch-build/pull/1968